.github/workflows/test.yml

```diff
@@ -25,8 +25,7 @@ jobs:
       env:
         EXECUTING_SLOW_TESTS: 1
       run: |
-        coverage run --include=executing/executing.py -m unittest tests.test_main
-        coverage run --include=executing/executing.py --append -m pytest tests/test_pytest.py
+        coverage run --include=executing/executing.py --append -m pytest tests
```

Comment:
At some point this needs to switch to including the whole executing folder, while accounting for the version stuff in `__init__`

Reply:
I don't understand what you mean here? 
There are no tests in the executing folder.

Reply:
Sorry, I see how it wasn't clear what I was talking about, and in any case it's not relevant to this PR. See #52.